### ISSUE TITLE
Dots in js.scope

### ISSUE THREAD

**Issue by vouillon** [alainfrisch/gen_js_api#160]
We used to be able to write:
```ocaml
module PlacesService: sig
  type t
  val new_places_service: Map.t -> t [@@js.new]
  ...
end
[@js.scope "google.maps.places"]
```
which was equivalent to using nested scopes `[@js.scope "places"][@js.scope "maps"][@js.scope "google"]`.

Since release 1.07, this no longer works since the following piece of code is generated:
```ocaml
Ojs.get_prop_ascii Ojs.global "google.maps.places"
```

Is this the expected behavior, or is this a bug?

**Comment by nojb** [comment]
cc @mlasson

**Comment by cannorin** [comment]
I created a PR which would allow us to write `[@js.scope ("google", "maps", "places")]`: #168

**Comment by mlasson** [comment]
Fixed by #168
